Jenkinsfile

```diff
@@ -48,7 +48,7 @@ pipeline {
     	    steps {
                 sh """
                 RESPONSE_CODE=\$(curl -o /dev/null -s -w "%{http_code}\n" http://$NGINX_DEV_IP)
-                if [ "\$RESPONSE_CODE" != 201 ]; then
+                if [ "\$RESPONSE_CODE" == 201 ]; then
```

Comment:
This is slightly incorrect, as the test you should be doing is checking if the status code is not equal to 200. The test you're running will only check if the response is 201 and only then give an error message. This will not take into account all other error messages. When you do the test `!= 200` this will give an error message whenever the status code is not 200 which is what we want.